src/modules/repository/implementation/sequelize/repositories/paranet-synced-asset-repository.js

```diff
@@ -30,6 +32,17 @@ class ParanetSyncedAssetRepository {
         });
     }
 
+    async getParanetSyncedAssetRecordsCountByDataSource(paranetUal, dataSource) {
```

Comment:
Would it make sense to get count instead of getting all data, if there is paranet with 20k record we need to get all of them turn them in objects and than just to get count.
What about using just
`SELECT COUNT(*) FROM ..